### DISCUSSION TITLE
Error: The cwd: /home/runner/work/hexo-src/hexo-src/github-pages-deploy-action-temp-deployment-folder does not exist!

### DISCUSSION THREAD

**Discussion by lozn00** [JamesIves/github-pages-deploy-action#1775] [Q&A]
Run JamesIves/github-pages-deploy-action@v4
    ╭━━━╮╭╮╭╮╱╭╮╱╱╭╮╱╱╭━━━╮
    ┃╭━╮┣╯╰┫┃╱┃┃╱╱┃┃╱╱┃╭━╮┃
    ┃┃╱╰╋╮╭┫╰━╯┣╮╭┫╰━╮┃╰━╯┣━━┳━━┳━━┳━━╮
    ┃┃╭━╋┫┃┃╭━╮┃┃┃┃╭╮┃┃╭━━┫╭╮┃╭╮┃┃━┫━━┫
    ┃╰┻━┃┃╰┫┃╱┃┃╰╯┃╰╯┃┃┃╱╱┃╭╮┃╰╯┃┃━╋━━┃
    ╰━━━┻┻━┻╯╱╰┻━━┻━━╯╰╯╱╱╰╯╰┻━╮┣━━┻━━╯
    ╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╭━╯┃
    ╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╰━━╯
    ╭━━━╮╱╱╱╱╱╭╮╱╱╱╱╱╱╱╱╭━━━╮╱╱╭╮
    ╰╮╭╮┃╱╱╱╱╱┃┃╱╱╱╱╱╱╱╱┃╭━╮┃╱╭╯╰╮
    ╱┃┃┃┣━━┳━━┫┃╭━━┳╮╱╭╮┃┃╱┃┣━┻╮╭╋┳━━┳━╮
    ╱┃┃┃┃┃━┫╭╮┃┃┃╭╮┃┃╱┃┃┃╰━╯┃╭━┫┃┣┫╭╮┃╭╮╮
    ╭╯╰╯┃┃━┫╰╯┃╰┫╰╯┃╰━╯┃┃╭━╮┃╰━┫╰┫┃╰╯┃┃┃┃
    ╰━━━┻━━┫╭━┻━┻━━┻━╮╭╯╰╯╱╰┻━━┻━┻┻━━┻╯╰╯
    ╱╱╱╱╱╱╱┃┃╱╱╱╱╱╱╭━╯┃
    ╱╱╱╱╱╱╱╰╯╱╱╱╱╱╱╰━━╯
    
    💖 Support: https://github.com/sponsors/JamesIves
    📣 Maintained by James Ives: https://jamesiv.es
    🚀 Getting Started Guide: https://github.com/JamesIves/github-pages-deploy-action
    ❓ Discussions / Q&A: https://github.com/JamesIves/github-pages-deploy-action/discussions
    🔧 Report a Bug: https://github.com/JamesIves/github-pages-deploy-action/issues
Checking configuration and starting deployment… 🚦
Deploying using Deploy Token… 🔑
Configuring git…
/usr/bin/git config --global --add safe.directory /home/runner/work/hexo-src/hexo-src
/usr/bin/git config user.name lozn00
/usr/bin/git config user.email qssq5[21](https://github.com/lozn00/hexo-src/actions/runs/12499243396/job/34874019266#step:9:22)@gmail.com
/usr/bin/git config core.ignorecase false
/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
/usr/bin/git remote rm origin
/usr/bin/git remote add origin https://x-access-token:***
***
***
***
***
***
***
***
***
***
***
***
***
***@github.com/webx.git
Git configured… 🔧
Starting to commit changes…
/usr/bin/git ls-remote --heads https://x-access-token:***
***
***
***
***
***
***
***
***
***
***
***
***
***@github.com/webx.git refs/heads/main
warning: url contains a newline in its path component: https://x-access-token:***
***
***
***
***
***
***
***
***
***
***
***
***
***@github.com/webx.git/
fatal: credential url cannot be parsed: https://x-access-token:***
***
***
***
***
***
***
***
***
***
***
***
***
***@github.com/webx.git/
fatal: remote helper 'https' aborted session
Running post deployment cleanup jobs… 🗑️
/usr/bin/git checkout -B github-pages-deploy-action/efrs98dq9
Error: The cwd: /home/runner/work/hexo-src/hexo-src/github-pages-deploy-action-temp-deployment-folder does not exist!
Notice: Deployment failed! ❌

**Comment by cjvanlissa** [comment]
I get the same https://github.com/cjvanlissa/worcs/actions/runs/21252608242/job/61158036301
